sentences/de/assist_satellite_HassBroadcast.yaml

```diff
@@ -0,0 +1,8 @@
+language: "de"
+intents:
+  HassBroadcast:
+    data:
+      - sentences:
+          - "((gib|gebe) (bekannt|durch)|lass alle (wissen|hören)) [dass ]{message}"
+          - "verkünde [überall ][dass ]{message}"
+          - "sag[e] überall [dass ]{message}"
```

Comment:
```suggestion
          - "((gib|gebe) (bekannt|durch)|lass alle (wissen|hören))[, dass] {message}"
          - "verkünde[ überall][, dass] {message}"
          - "sag[e] überall[, dass] {message}"
          - "gib[ überall] Bescheid[, dass] {message}"
```

Reply:
I like the "gib überall Bescheid" addition but why add the comma? isn't punctuation from STT stripped before being handed over to intent matching?

Reply:
> isn't punctuation from STT stripped before being handed over to intent matching?

mhhh ... good question :thinking: not sure :see_no_evil:

Reply:
just verified that it is indeed stripped. then again one could argue that's why it's okay to have the punctuation in there :) 

![image](https://github.com/user-attachments/assets/631f18e2-6f8f-46ac-b9b1-50587381f69b)